src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -699,6 +707,15 @@ public async Task UploadValues_Success(Uri echoServer)
             byte[] result = await UploadValuesAsync(wc, echoServer.ToString(), new NameValueCollection() { { "Data", ExpectedText } });
             Assert.Contains(ExpectedTextAfterUrlEncode, Encoding.UTF8.GetString(result));
         }
+
+        private static void AddMD5Header(WebClient wc, string data)
```

Comment:
Nit: I think the implicit conversion to UTF-8 in this method is potentially confusing. Maybe change the name, or just add a comment to make that clear?

Reply:
Thx.  I'll add a comment to clarify the use of UTF-8.